.github/renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":approveMajorUpdates",
+        ":semanticCommitScopeDisabled"
+    ],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "7 days",
+
+    "packageRules": [
+        {
+            "description": "Automerge minor version updates. Major versions must be approved from the Dependency Dashboard issue, because we use the :approveMajorUpdates preset.",
+            "automerge": true,
+            "matchUpdateTypes": ["minor"]
+        },
+        {
+            "description": "Ignore patch versions to reduce noise.",
+            "enabled": false,
+            "matchUpdateTypes": ["patch"]
+        },
```

Comment:
Is ignoring patch versions a common practice? We mostly use major ranges (`^`) and I guess we want automatic in-range dependency updates mostly to make sure that bug fixes are included. From that perspective, making minor updates but ignoring patch updates doesn't make sense.

Reply:
It's not common to ignore them, and I'd recommend against that - they're usually lower risk bug fixes. What you _can_ do is group and schedule patch updates so that you get e.g. 1 PR per week

Reply:
👍 removing this. By principle of least resistance I haven't added anything around grouping, but certainly can if asked to.